example/src/CustomReactionsV2.tsx

```diff
@@ -12,7 +12,6 @@ export default function CustomReactions() {
   const [emojis, increment] = useEmojis(DEFAULT_EMOJI_OPTIONS);
 
   const selected = emojis
-    .sort((a, b) => (a.emoji < b.emoji ? 1 : -1))
```

Comment:
Can the devs who choose to use the custom blocks sort the emojis however they want? Maybe some of them wants to sort them by their counters instead of having them in fixed position. I need to look into this

Reply:
you could still easily have a custom sort if you want.

Reply:
I tried and it works, only problem is that I can't make an easy sort with the counters because they might be undefined and typescript won't allow that. But that's not a problem with this code at all